From: Carlos Garnacho <carlosg@gnome.org>
Date: Mon, 26 May 2025 17:27:17 +0200
Subject: build: Bump Mutter ABI version

We didn't do so for GNOME 49, so do it now.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 9f77e34..1ba0a51 100644
--- a/meson.build
+++ b/meson.build
@@ -10,7 +10,7 @@ split_version = meson.project_version().split('.')
 # the Cogl and Clutter libraries also use this API version.
 # The API version is increased each development cycle,
 # starting with 0 in 3.23.x
-mutter_api_version = '16'
+mutter_api_version = '17'
 
 clutter_pc = 'mutter-clutter-' + mutter_api_version
 cogl_pc = 'mutter-cogl-' + mutter_api_version
